src/smart-components/role/add-role/inventory-groups-role.js

```diff
@@ -133,78 +148,87 @@ const InventoryGroupsRole = (props) => {
-                  loadingVariant: {
-                    text: intl.formatMessage(messages.seeMore),
-                    onClick: () => {
-                      fetchData([permissionId], { page: state[permissionId].page + 1, name: state[permissionId].filterValue });
-                      dispatchLocally({ type: 'setPage', key: permissionId, page: state[permissionId].page++ });
+                onToggle={(isOpen) => {
+                  !isOpen && state[permission].filterValue?.length > 0 && fetchData([permission]);
+                  dispatchLocally({ type: 'toggle', key: permission, filterValue: '', page: 1, isOpen });
+                }}
+                onClear={() => clearSelection(permission)}
+                onFilter={(event) => {
+                  if (event) {
+                    dispatchLocally({ type: 'setFilter', key: permission, filterValue: event.target.value });
+                    debouncedFetch(() => fetchData([permission], { name: state[permission].filterValue }), 2000);
+                  }
+                }}
+                isOpen={state[permission].isOpen}
+                hasInlineFilter
+                {...(!isLoading &&
+                  resourceTypes[permission] &&
+                  Object.values(resourceTypes[permission]).length < totalCount && {
+                    loadingVariant: {
+                      text: intl.formatMessage(messages.seeMore),
+                      onClick: () => {
+                        fetchData([permission], { page: state[permission].page + 1, name: state[permission].filterValue });
+                        dispatchLocally({ type: 'setPage', key: permission, page: state[permission].page++ });
+                      },
                     },
-                  },
-                })}
-              {...(isLoading && { loadingVariant: 'spinner' })}
-            >
-              {[
-                ...(options?.length > 0
-                  ? [<SelectOption key={`${permissionId}-all`} value={intl.formatMessage(messages.selectAll, { length: options?.length })} />]
-                  : []),
-                ...(options?.map((option, index) => <SelectOption key={`${permissionId}-${index + 1}`} value={option?.name} />) || []),
-              ]}
-            </Select>
-          </Tooltip>
-        </GridItem>
+                  })}
+                {...(isLoading && { loadingVariant: 'spinner' })}
+              >
+                {[
+                  ...(options?.length > 0
+                    ? [<SelectOption key={`${permission}-all`} value={intl.formatMessage(messages.selectAll, { length: options?.length })} />]
+                    : []),
+                  ...(options?.map((option, index) => <SelectOption key={`${permission}-${index + 1}`} value={option?.name} />) || []),
+                ]}
+              </Select>
+            </Tooltip>
+          </GridItem>
+          <GridItem lg={2} md={4} sm={2} className="rbac-m-hide-on-sm">
```

Comment:
```suggestion
          <GridItem lg={2} md={4} sm={2}>
```
do we need to hide this button on small devices, the space is still reserved, but just blank now